ios/browser/api/ads/brave_ads.mm

```diff
@@ -18,6 +18,7 @@
 #include "base/task/thread_pool.h"
 #include "base/task_runner_util.h"
 #include "bat/ads/ad_event_history.h"
+#include "bat/ads/ad_history_info.h"
```

Comment:
Is this needed? Was it removed from an already included header?

Reply:
It was removed as we now forward declare in the other header